src/main/scala/no/ndla/audioapi/controller/InternController.scala

```diff
@@ -37,5 +37,9 @@ trait InternController {
       }
     }
 
+    get("/external/:external_id") {
```

Comment:
Når article-api importerer noder vil den spørre audio-api om vi har en audio fil med en ekstern id (node id fra gamle NDLA). Hvis vi har den returnerer denne metadata for audio filen.